README.rst

```diff
@@ -141,6 +141,15 @@ Please note the following guidelines for contributing:
   * If you are adding new functionality, you must include basic tests
     and documentation.
 
+
```

Comment:
I understand it would have been useful, I just don't think it's up to every single Python package to document how setuptools works. 

BTW the new change is incorrect, there's no requirements.txt file to reference.